doc-content/enterprise-only/project-data/proc-define-local-variables.adoc

```diff
@@ -0,0 +1,28 @@
+[id='proc-define-local-variables_{context}']
+= Defining local variables
+
+Local variables are available within their process element, such as an activity. They are initialized when the element context is initialized, that is, when the execution workflow enters the node and execution of the `onEntry` action has finished, if applicable. They are destroyed when the element context is destroyed, that is, when the execution workflow leaves the element.
+
+Values of local variables can be mapped to global or process variables. This enables you to maintain relative independence of the parent element that accommodates the local variable. Such isolation might help prevent technical exceptions.
+
+A local variable is a variable that exists in a child element context of a process and can be accessed only from within this context. Local variables belong to the particular element of a process.
+
+For tasks, with the exception of the Script task, you can define [property]``Data Input Assignments`` and [property]``Data Output Assignments`` in the *Assignments* property. Data Input Assignment defines variables that enter the Task and therefore provide the entry data needed for the task execution. The Data Output Assignments can refer to the context of the Task after execution to acquire output data.
+
+User Tasks present data related to the actor that is executing the User Task. Additionally, User Tasks also request the actor to provide result data related to the execution.
+
+To request and provide the data, use task forms and map the data in the Data Input Assignment parameter to a variable. Map the data provided by the user in the Data Output Assignment parameter if you want to preserve the data as output.
+
+.Prerequisites
+* You have created a project in {CENTRAL} and it contains at least one business process asset that has at least one task that is not a script task.
+
+.Procedure
+. Open a business process asset.
+. Select a task that is not a script task.
+. Click the *Properties* icon on the upper-right side of the screen to open the *Properties* panel.
+. If necessary, expand the *Data Assignments* section.
```

Comment:
Where is this section located? When I open the Properties panel in the jBPM 7.39.0.Final distribution (UI should be the same as RHPAM) there is just the Implementation/Execution section.
![obrázok](https://user-images.githubusercontent.com/6797884/86268719-e51da500-bbc8-11ea-86c4-3d1838b24b14.png)

Reply:
I believe it was in UI earlier, since I did not change the content but only moved the content to a new module. However, I removed the Data Assignments step.